rollup.config.js

```diff
@@ -8,7 +9,7 @@ import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
 import pkg from './package.json';
 
 const input = './compiled/index.js';
-const external = id => !id.startsWith('.') && !id.startsWith('/');
+const external = id => !id.startsWith('.') && !path.isAbsolute(id);
```

Comment:
what's this do?

Reply:
This is from PR #1353 , which I need to run the project. It'll go away when I merge master